### PR TITLE
Closes #2060 by updating zrpc and zrpcunsafe logging

### LINKED ISSUE #2060: Review zrpc category log statements and consider moving some to zrpcunsafe category

### PR DESCRIPTION
**Pull request by bitcartel**

### CODE REVIEW

**Comment by bitcartel** [comment]
Reviewers, here's what the logging now looks like.
https://gist.github.com/bitcartel/8f1c909c020ce4e076e22aeddaaf98d1

File: src/init.cpp [hunk 1]
```diff
@@ -802,6 +802,15 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
     if (GetBoolArg("-nodebug", false) || find(categories.begin(), categories.end(), string("0")) != categories.end())
         fDebug = false;
 
+    // Special case: if debug=zrpcunsafe, implies debug=zrpc, so add it to debug categories
+    if (find(categories.begin(), categories.end(), string("zrpcunsafe")) != categories.end()) {
+        if (find(categories.begin(), categories.end(), string("zrpc")) == categories.end()) {
+            LogPrintf("%s: parameter interaction: -debug=zrpcunsafe -> -debug=zrpc\n", __func__);
```
Comment (str4d) [inline src/init.cpp]: `"%s: parameter interaction: -debug=zrpcunsafe -> setting -debug=zrpc\n"` for parity with other log messages. Non-blocking.

File: src/wallet/asyncrpcoperation_sendmany.cpp [hunk 2]
```diff
@@ -276,13 +276,13 @@ bool AsyncRPCOperation_sendmany::main_impl() {
...
 
-    LogPrint("zrpc", "%s: spending %s to send %s with fee %s\n",
+    LogPrint((isfromtaddr_) ? "zrpc" : "zrpcunsafe", "%s: spending %s to send %s with fee %s\n",
             getId(), FormatMoney(targetAmount, false), FormatMoney(sendAmount, false), FormatMoney(minersFee, false));
-    LogPrint("zrpc", "%s: -  transparent input: %s (to choose from)\n", getId(), FormatMoney(t_inputs_total, false));
-    LogPrint("zrpc", "%s: -      private input: %s (to choose from)\n", getId(), FormatMoney(z_inputs_total, false));
-    LogPrint("zrpc", "%s: - transparent output: %s\n", getId(), FormatMoney(t_outputs_total, false));
-    LogPrint("zrpc", "%s: -     private output: %s\n", getId(), FormatMoney(z_outputs_total, false));
-    LogPrint("zrpc", "%s: -                fee: %s\n", getId(), FormatMoney(minersFee, false));
+    LogPrint((isfromtaddr_) ? "zrpc" : "zrpcunsafe", "%s: transparent input: %s (to choose from)\n", getId(), FormatMoney(t_inputs_total, false));
```
Comment (daira) [inline src/wallet/asyncrpcoperation_sendmany.cpp]: Why is this unsafe in the ``!isfromtaddr_`` case?

File: src/init.cpp (hunk 1 above)
Comment (bitcartel) [inline src/init.cpp]: Done

File: src/wallet/asyncrpcoperation_sendmany.cpp (hunk 2 above)
Comment (bitcartel) [inline src/wallet/asyncrpcoperation_sendmany.cpp]: Should be zrpc.  Updated.

**Comment by bitcartel** [comment]
Minor changes to address comments in review complete.  Force pushed.

**Comment by bitcartel** [comment]
@zkbot r+

**Pull request merged**